app/src/main/java/com/sdp/movemeet/backend/serialization/UserSerializer.java

```diff
@@ -15,6 +16,10 @@
     public static final String PHONE_KEY = "phone";
     // The key used to access the description attribute of a serialized User
     public static final String DESCRIPTION_KEY = "description";
+    // The key used to access the created activities attribute of a serialized User
+    private static  final String CREATED_ACTIVITIES_KEY = "createdActivity";
```

Comment:
Nice! Thank you for taking our conversation into account and adding this field in the `User` objects! This way, as suggested a few weeks ago by @maximilian1064, we will be able to not only completely delete the user from Firebase but also the "Firebase Firestore documents" of the activities he created as an organiser! The "delete account" feature will hence make much more sense!